images/linux/scripts/installers/2004/php.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+################################################################################
+##  File:  php.sh
+##  Desc:  Installs php
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/etc-environment.sh
+source $HELPER_SCRIPTS/document.sh
+
+LSB_RELEASE=$(lsb_release -rs)
+
+set -e
+
+apt-add-repository ppa:ondrej/php -y
+
+# Install php7.1
+apt-fast install -y --no-install-recommends \
+    php7.1 \
+    php7.1-bcmath \
+    php7.1-bz2 \
+    php7.1-cgi \
+    php7.1-cli \
+    php7.1-common \
+    php7.1-curl \
+    php7.1-dba \
+    php7.1-dev \
+    php7.1-enchant \
+    php7.1-fpm \
+    php7.1-gd \
+    php7.1-gmp \
+    php7.1-imap \
+    php7.1-interbase \
+    php7.1-intl \
+    php7.1-json \
+    php7.1-ldap \
+    php7.1-mbstring \
+    php7.1-mcrypt \
+    php7.1-mysql \
+    php7.1-odbc \
+    php7.1-opcache \
+    php7.1-pgsql \
+    php7.1-phpdbg \
+    php7.1-pspell \
+    php7.1-readline \
+    php7.1-recode \
+    php7.1-snmp \
+    php7.1-soap \
+    php7.1-sqlite3 \
+    php7.1-sybase \
+    php7.1-tidy \
+    php7.1-xml \
+    php7.1-xmlrpc \
+    php7.1-xsl \
+    php7.1-zip
+apt-get remove --purge -yq php7.1-dev
+
+# Install php7.2
+apt-fast install -y --no-install-recommends \
+    php7.2 \
```

Comment:
it seems to me that the set of PHP packages we are installing is almost the same for all versions. 
Does it makes sense to create a list of PHP versions and iterate over it to install common packages? and then install specific packages
something like
```
php_versions=("7.1" "7.2" "7.3" "7.4")
for version in php_versions; do
    php${vesrion}\
    php${vesrion}-bcmath \
    php${vesrion}-bz2 \
    php${vesrion}-cgi \
    ...
done;

#specific packages installation below
...
```

Reply:
I like this idea but perhaps as a separate cleanup item so it doesn't block this OS.

Reply:
@AlenaSviridenko, Could you please review?

Reply:
@al-cheb new php.sh is beautiful, thanks!